src/Core/Components/Nav/FluentNavCategory.razor.cs

```diff
@@ -0,0 +1,303 @@
+// ------------------------------------------------------------------------
+// This file is licensed to you under the MIT License.
+// ------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Components;
+using Microsoft.FluentUI.AspNetCore.Components.Extensions;
+using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+using Microsoft.JSInterop;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// Represents a group of related navigation items for use in a fluent navigation interface.
+/// </summary>
+/// <remarks>Use this class to organize navigation elements into logical groups when building fluent or
+/// hierarchical navigation structures. Grouping navigation items can improve usability and clarity in user interfaces
+/// that support complex navigation scenarios.</remarks>
+public partial class FluentNavCategory : FluentComponentBase, IAsyncDisposable
```

Comment:
Since you inherit from `FluentComponentBase` you don't need to specify `IAsyncDisposable` here again